docs/build/index.md

```diff
@@ -0,0 +1,8 @@
+<a id='Main.Lapidary-1'></a>
+
+<a href='#Main.Lapidary-1'> # </a>**Constant**
+
+A documentation generator for Julia.
```

Comment:
The alternative is to stick a copy of the README in here (Note: the `index.md` is required on mkdocs). I definitely like how that's generated in the rewrite branch!